src/xma/include/lib/xmahw_lib.h

```diff
@@ -245,7 +234,7 @@ typedef struct XmaHwKernel
   XmaHwKernel() {
    std::memset(name, 0, sizeof(name));
     in_use = false;
-    context_opened = false;
+    context_opened = true;
```

Comment:
We should be able to delete lot of the below variables. It can be in next PR, but please take a note.

Reply:
Noted.

Reply:
Noted